src/components/SignInPage.css

```diff
@@ -0,0 +1,50 @@
+.header-content {
+  position: relative;
+}
+
+.signIn-headline {
+  position: absolute;
+  top: -3px;
+  left: 5px;
+  z-index: 1;
+  font-size: 105px;
+  font-family: "Pegasus";
+  color: var(--dark-color);
+}
+
+.signIn-image {
+  display: flex;
+  align-items: center;
+  margin: 15px;
+  height: 55vh;
+  border-radius: 4px;
+  opacity: 0.8;
+  background-position: -1000px -600px;
+  background-image: url(../img/signInBackground.png);
+}
+
+.signIn-input {
+  display: block;
+  margin: 0 auto;
+  width: 270px;
+  height: 64px;
+  border: solid 2px transparent;
+  border-radius: 4px;
+  background: var(--background-color);
+  opacity: 0.5;
+  text-align: center;
+  font-size: 21px;
+  color: var(--dark-color);
+}
```

Comment:
I see many fixed widths (some to pixels, some to vh/vh)
I'm not sure what you want to achieve here, but maybe there is a simpler way.